src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -260,14 +275,41 @@ int halide_hexagon_remote_set_performance_mode(int mode) {
     mipsTotal = mipsPerThread * 2;
 
     bwMegabytesPerSec = bwBytePerSec >> 20;
-    return halide_hexagon_remote_set_performance(set_mips,
-                                                 mipsPerThread,
-                                                 mipsTotal,
-                                                 set_bus_bw,
-                                                 bwMegabytesPerSec,
-                                                 busbwUsagePercentage,
-                                                 set_latency,
-                                                 latency);
+
+    memset(&request, 0, sizeof(HAP_power_request_t));
+    request.type = HAP_power_set_apptype;
+    request.apptype = HAP_POWER_COMPUTE_CLIENT_CLASS;
+    retval = HAP_power_set(NULL, &request);
+    if (0 != retval) {
+        log_printf("HAP_power_set(HAP_power_set_apptype) failed (%d)\n", retval);
+        return -1;
+    }
+
+    memset(&request, 0, sizeof(HAP_power_request_t));
+    request.type = HAP_power_set_DCVS_v2;
+    request.dcvs_v2.dcvs_enable = TRUE;
+    request.dcvs_v2.dcvs_option = HAP_power_dcvs_v2_payload::HAP_DCVS_V2_POWER_SAVER_MODE;
+    request.dcvs_v2.set_dcvs_params = TRUE;
+    request.dcvs_v2.dcvs_params.min_corner = HAP_DCVS_VCORNER_DISABLE;
+    request.dcvs_v2.dcvs_params.max_corner = HAP_DCVS_VCORNER_DISABLE;
+    request.dcvs_v2.dcvs_params.target_corner = voltage_corner[mode];
+    request.dcvs_v2.set_latency = set_latency;
+    request.dcvs_v2.latency = latency;
+    retval = HAP_power_set(NULL, &request);
+    if (0 == retval) {
+        log_printf("HAP_power_set(HAP_power_set_dcvs_v2) succeeded (%d)\n", retval);
```

Comment:
I think printing in the non-error case could have significant performance implications, it's also just spammy.

Reply:
ok. I will remove it.